lib/passcode_screen.dart

```diff
@@ -73,15 +77,15 @@ class _PasscodeScreenState extends State<PasscodeScreen> with SingleTickerProvid
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      backgroundColor: Colors.black.withOpacity(0.8),
+      backgroundColor: widget.backgroundColor ?? Colors.black.withOpacity(0.8),
```

Comment:
Please change it to be a default background color instead,  the same way you done it for a title color

Reply:
Your default background color is not Const, there're withOpacity function, therefore it will not be available compile time

Reply:
you are right